src/components/task/details.cjsx

```diff
@@ -0,0 +1,23 @@
+React = require 'react'
+BS = require 'react-bootstrap'
+Time = require '../time'
+ArbitraryHtml = require '../html'
+
+module.exports = React.createClass
+  render: ->
+    {model} = @props
+
+    if model.description_html
+      detailPopover =
+        <BS.Popover title='Instructions' className='-task-details-popover task-details-popover'>
+          <ArbitraryHtml html={model.description_html}/>
```

Comment:
I don't think we're allowing arbitrary HTML in the descriptions; just text with newlines. I'm not sure how to handle text wrapping with newlines. Maybe there is some CSS to respect newlines (like `<pre>`) but still allow the text to wrap?